db/db_utils.go

```diff
@@ -109,90 +109,6 @@ func InsertManyUnordered(ctx context.Context, collection string, items ...any) e
 	return errors.Wrapf(errors.WithStack(err), "inserting unordered documents")
 }
 
-// CreateCollections ensures that all the given collections are created,
```

Comment:
I moved these to the bottom of the file to have some separation of test only vs actual queried functions. I'll create a ticket to move them out of this package/file